ApplyTMTemplate/Sdl.Community.ApplyTMTemplate/ViewModels/MainWindowViewModel.cs

```diff
@@ -446,7 +479,7 @@ private bool IsThereAnyTmSelected()
 			return TmCollection.Any(tm => tm.IsSelected);
 		}
 
-		public bool CanExecuteApply => _templateValidWithResources && IsThereAnyTmSelected();
+		public bool CanExecuteApply => (int)_templateValidity > 1 && IsThereAnyTmSelected();
```

Comment:
Use Int32.TryParse()
https://docs.microsoft.com/en-us/dotnet/api/system.int32.tryparse?view=netframework-4.8

Reply:
I read about the enums that the best practice is to specify a numeric equivalent to each of its values and to use direct casting to int when that numerical equivalent is needed.